Mothercell/Hive.IO/Hive.IO/EnergySystems/Emitter.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hive.IO.EnergySystems
+{
+    public class Radiator : Emitter
+    {
+        public Radiator(double investmentCost, double embodiedGhg, bool isHeating, bool isCooling, double inletTemperature, double returnTemperature) 
+            : base(investmentCost, embodiedGhg, isHeating, isCooling)
+        {
+
+            double [] inletTempArray = new double[Horizon];
```

Comment:
you don't need to call it an _array_. the IDE already tells you that - call it `inletTemperatures` instead.